plugins/WebOSMetaPlugin/index.js

```diff
@@ -163,10 +164,10 @@ class WebOSMetaPlugin {
 			compilation.hooks.webosMetaLocalizedAppinfo = new SyncWaterfallHook(['appinfo', 'details']);
 
 			// Hook into html-webpack-plugin to dynamically set page title
-			if (compilation.hooks.htmlWebpackPluginBeforeHtmlGeneration) {
-				compilation.hooks.htmlWebpackPluginBeforeHtmlGeneration.tapAsync(
-					'WebOSMetaPlugin',
-					(params, callback) => {
+			if (this.htmlWebpackPlugin) {
+				this.htmlWebpackPlugin
+					.getHooks(compilation)
+					.beforeAssetTagGeneration.tapAsync('WebOSMetaPlugin', (params, callback) => {
```

Comment:
Would you make this form as same as the similar part in Prerendering Plugin?
```suggestion
				this.htmlWebpackPlugin
					.getHooks(compilation)
					.beforeAssetTagGeneration.tapAsync('WebOSMetaPlugin', (htmlPluginData, callback) => {
```